zk-states/src/types.ts

```diff
@@ -128,8 +128,20 @@ interface ProofSuccessUpdate {
   updateType: "proofSuccess";
   callId: string;
 }
+export type initializationProgress =
+  | "initializing"
```

Comment:
```suggestion
  | "initializing"
```
this first step is redundant since everything before compilation isn't really time consuming